testdata/goldens/existing_test_file_with_package_level_comments.go

```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The Go Authors. All rights reserved.
```

Comment:
Please update these test comments to some Lorum Ipsum. We probably shouldn't have real comments mixed in our test cases.